docs/vsphere_ci/windows-server-2022.pkr.hcl

```diff
@@ -78,8 +83,7 @@ source "vsphere-iso" "windows-server-2022" {
   insecure_connection  = "true"
   iso_paths            = ["${var.os-iso-path}", "${var.vmtools-iso-path}"]
   network_adapters {
-    network      = "dev-segment"
-    network_card = "vmxnet3"
```

Comment:
Is there a reason to drop this? You might default to a non-paravirtualized network device which will result in reduced network performance which can be important when pulling images.

Reply:
Nice catch, I had dropped it when I was using a different (incorrect) network with a different network adapter and forgot to add it back in